{"name": "Test of failed attempt to login with the wrong fields", "status": "passed", "description": "Negative test of the endpoint \"Авторизация пользователя\" POST /api/auth/login.Parameterized test, checks impossibility of authorization when any of the mandatory fields is wrong", "parameters": [{"name": "wrong_field", "value": "'password'"}], "start": 1719670162494, "stop": 1719670162632, "uuid": "d78577c3-f2b6-4197-a872-e8b72a27f5e8", "historyId": "b2d0688b28b6c26e31ec650bdc846a8b", "testCaseId": "9ec928b196acec01e210e78e4d76d55b", "fullName": "tests.test_user_login.TestUserLogin#test_login_with_any_wrong_field_failed", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "DESKTOP-4KK18J8"}, {"name": "thread", "value": "600-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}